src/components/measureRoutes/MeasureRoutes.tsx

```diff
@@ -1,30 +1,37 @@
 import React from "react";
-import { Route, Switch, BrowserRouter, Redirect } from "react-router-dom";
+import {
+  Route,
+  Navigate,
+  RouterProvider,
+  createBrowserRouter,
+  createRoutesFromElements,
+} from "react-router-dom";
 import EditMeasure from "../editMeasure/EditMeasure";
 import MeasureLanding from "../measureLanding/MeasureLanding";
 import TimeoutHandler from "./TimeoutHandler";
 import NotFound from "../notfound/NotFound";
 
-export function MeasureRoutes() {
-  return (
-    <>
-      <Switch>
-        <Route exact path="/measures" component={MeasureLanding} />
-        <Route path="/measures/:id/edit" component={EditMeasure} />
-        <Route path="/404" component={NotFound} />
-        <Redirect to="/404" />
-      </Switch>
-    </>
-  );
-}
+export const routesConfig = [
+  {
+    children: [
+      { path: "/", element: <MeasureLanding /> },
+      { path: "/measures", element: <MeasureLanding /> },
+      { path: "/measures/*", element: <MeasureLanding /> },
+      { path: "/measures/:id/edit", element: <EditMeasure /> },
+      { path: "/measures/:id/edit/*", element: <EditMeasure /> },
```

Comment:
This is the parent route, Do we need these 2 extra routes ?
`{ path: "/measures/*", element: <MeasureLanding /> },`
and 
`{ path: "/measures/:id/edit/*", element: <EditMeasure /> },`

Reply:
Not both. 
First we don't need.

{ path: "/measures/:id/edit/*", element: <EditMeasure /> }, we need as the wild card operator tells router to load editMeasure for all subRoutes after edit to include details.

However we can remove 
      { path: "/measures/:id/edit", element: <EditMeasure /> },

removing